requirements-test.txt

```diff
@@ -9,3 +9,4 @@ freezegun
 django-extensions
 openwisp-utils[qa]
 pylinkvalidator
+djangosaml2~=1.2.0
```

Comment:
why don't we avoid this duplicated requirement and modify the github action workflow to do `pip install -e .[saml]` instead of `pip install -e .`?